include/psen_scan_v2/diagnostics.h

```diff
@@ -166,13 +171,12 @@ class MonitoringFrameDiagnosticMessage
   ErrorLocation error_location_;
 };
 
-constexpr inline MonitoringFrameDiagnosticMessage::MonitoringFrameDiagnosticMessage(const ScannerId& id,
-                                                                                    const ErrorLocation& location)
+constexpr inline Message::Message(const ScannerId& id, const ErrorLocation& location)
   : id_(id), error_location_(location)
 {
 }
 
-constexpr inline bool MonitoringFrameDiagnosticMessage::operator==(const MonitoringFrameDiagnosticMessage& rhs) const
+constexpr inline bool Message::operator==(const Message& rhs) const
```

Comment:
Code inconsistent.
```suggestion
constexpr inline bool diagnostic::Message::operator==(const diagnostic::Message& rhs) const
```

Reply:
I get a compiler error message when I do this:

error: extra qualification ‘psen_scan_v2::monitoring_frame::diagnostic::Message::’ on member ‘operator==’ [-fpermissive]